course/utils.py

```diff
@@ -1099,4 +1115,16 @@ def will_use_masked_profile_for_email(recipient_email):
             return True
     return False
 
+
+def get_course_specific_langs_choices():
+    # type: () -> Tuple[Tuple[str, Any], ...]
+    from django.conf import settings
+    return (("", _("(None)")), ) + tuple((k, _(v)) for (k, v) in settings.LANGUAGES)
+
+
+def get_available_languages():
+    # type: () -> List[Text]
+    from django.conf import settings
+    return [lang[0] for lang in settings.LANGUAGES]
```

Comment:
Would prefer `[lang_code for lang_code, descr in settings.LANGUAGES]`.

Reply:
Done.